### ISSUE TITLE
Host images on Cloudinary

### ISSUE THREAD

**Issue by parth-io** [usdevs/usc-website-hackathon-frontend#101]

**Comment by zsh-eng** [comment]
@parth-io Is this still an open issue?

**Comment by parth-io** [comment]
Yep, because currently IGs have to send their images to the developers to upload their images to Cloudinary. This issue should be titled more appropriately, "Add image upload widget to form."
